src/actions/homophones.py

```diff
@@ -0,0 +1,26 @@
+from talon import actions, app
+from .get_texts import get_texts
+
+
+def run_homophones_action(targets: dict):
+    """Replaced target with next homophone"""
+    texts = get_texts(targets, show_decorations=False)
+    try:
+        updated_texts = list(map(
+            lambda text: get_next_homophone(text),
+            texts
+        ))
+    except LookupError as e:
+        app.notify(str(e))
+        return
+    actions.user.cursorless_replace(
+        targets, updated_texts
+    )
+
+
+def get_next_homophone(word: str):
+    homophones = actions.user.homophones_get(word)
```

Comment:
This is the action that needs to be in knausj?

Reply:
yes one of them. I've outlined all the needed changes in the description of the pr

Reply:
Right yeah prob should have read that before I started reviewing 🤦‍♂️